components/brave_shields/common/brave_shields_panel.mojom

```diff
@@ -26,5 +26,16 @@ interface PanelHandler {
 };
 
 // WebUI-side handler for requests from the browser.
-interface Page {
+interface UIHandler {
+  OnSiteDataChanged(CurrentSite current_site);
+};
+
+interface DataHandler {
+  RegisterUIHandler(pending_remote<UIHandler> ui_handler);
+  GetCurrentSiteData() => (CurrentSite current_site);
+};
+
+struct CurrentSite {
+  url.mojom.Url location;
+  int32 total_blocked_resources;
```

Comment:
@petemill @simonhong does int32 feel excessive for total blocked count?

Reply:
I think it's fine and it's matched with the return value of `BraveShieldsDataController::GetTotalBlockedCount()` .